src/dialogs/xrefsdialog.cpp

```diff
@@ -128,11 +133,8 @@ void XrefsDialog::on_toTreeWidget_itemSelectionChanged()
     QString offset = item->text(0);
     ui->previewTextEdit->setPlainText( this->main->core->cmd("pdf @ " + offset).trimmed() );
     ui->previewTextEdit->moveCursor(QTextCursor::End);
-    int pos = ui->previewTextEdit->find( this->normalizeAddr(offset), QTextDocument::FindBackward);
+    // Again, does it make any sense?
+    // Also, this code should be refactored and shared instead of copied & pasted
```

Comment:
This is not radare. This is Iaito. Also, it doesn't make sense not to use comments where it makes sense.